tests/conftest.py

```diff
@@ -206,9 +205,6 @@ def get_valid_image():
 
 def verify_sane_test_environment():
     # check if required tools are in PATH, add any other checks here
-    if distutils.spawn.find_executable("mender-stress-test-client") is None:
-        raise SystemExit("mender-stress-test-client not found in PATH")
```

Comment:
This one got me so many times (my path is not `~/go/bin`)!

Reply:
This was the exact trigger for this PR, actually. I was getting annoyed to require such binary in my PATH for a test that seemed to be obsolete...